Missing env param when building on arm

### Summary

When im building on arm (armbian) im facing the problem 

 `CMake was unable to find a build program corresponding to "Ninja"`
Described in 
 - https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/BuildInstructionsLadybird.md#unable-to-find-a-build-program-corresponding-to-ninja

VCPKG log says
`Environment variable VCPKG_FORCE_SYSTEM_BINARIES must be set on arm, s390x, ppc64le and riscv platforms.`

 in this case simply an env param is missing

`export VCPKG_FORCE_SYSTEM_BINARIES=1`

### Operating system

Linux

### Steps to reproduce

1. Try build ladybord on armbian or another arm system


### Expected behavior

Run build

### Actual behavior

Runs into message 

 - CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.


### URL for a reduced test case

N/A

### HTML/SVG/etc. source for a reduced test case

```html
N/A
```


### Log output and (if possible) backtrace


```shell
CMake Error at Build/vcpkg/scripts/buildsystems/vcpkg.cmake:904 (message):
  vcpkg install failed.  See logs for more information:
  /home/omdm/git/ladybird/Build/release/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/CMakeDetermineSystem.cmake:124 (include)
  CMakeLists.txt:15 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
-- Configuring incomplete, errors occurred!
ninja: Entering directory `/home/omdm/git/ladybird/Build/release'
ninja: error: loading 'build.ninja': No such file or directory
./Meta/ladybird.sh: line 217: /home/omdm/git/ladybird/Build/release/bin/Ladybird: No such file or directory






Environment variable VCPKG_FORCE_SYSTEM_BINARIES must be set on arm, s390x, ppc64le and riscv platforms.
```


### Screenshots or screen recordings

_No response_

### Build flags or config settings

_No response_

### Contribute a patch?

- [ ] I’ll contribute a patch for this myself.

Same on Arch arm 
`==> Starting build()...
-- Determined host VCPKG_TARGET_TRIPLET: arm64-linux
-- Bootstrapping vcpkg before install
-- Bootstrapping vcpkg before install - failed
CMake Error at /home/markon/.cache/yay/ladybird/src/vcpkg/scripts/buildsystems/vcpkg.cmake:904 (message):
  vcpkg install failed.  See logs for more information:
  /home/markon/.cache/yay/ladybird/src/build/vcpkg-bootstrap.log
Call Stack (most recent call first):
  /usr/share/cmake/Modules/CMakeDetermineSystem.cmake:146 (include)
  CMakeLists.txt:18 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
==> ERROR: A failure occurred in build().
`

--- a/Meta/ladybird.py
+++ b/Meta/ladybird.py
@@ -307,10 +307,13 @@ def configure_build_env(platform: Platform, preset: str, jobs: Optional[str] = N
     if jobs:
         os.environ["VCPKG_MAX_CONCURRENCY"] = jobs
 
-    if platform.host_architecture == HostArchitecture.riscv64:
-        # vcpkg refuses to build ports on riscv64 and other less common architectures without this flag.
-        # With it set vcpkg will use the system provided CMake and Ninja binaries but will still download,
-        # build and use its own pinned versions of gn, meson and pkg-config.
+    needs_system_binaries = platform.host_architecture == HostArchitecture.riscv64 or (
+        platform.host_system == HostSystem.Linux and platform.host_architecture == HostArchitecture.AArch64
+    )
+    if needs_system_binaries:
+        # vcpkg does not ship prebuilt tool binaries for riscv64 or arm64 Linux and refuses to build
+        # ports on these hosts without this flag. With it set vcpkg uses the system provided CMake and
+        # Ninja binaries but still downloads, builds and uses its own pinned gn, meson and pkg-config.
         os.environ["VCPKG_FORCE_SYSTEM_BINARIES"] = "1"
 
     if "XDG_CACHE_HOME" not in os.environ:

